core/whistle-1.0.0/src/wh_util.erl

```diff
@@ -632,6 +633,12 @@ spawn(Module, Function, Arguments) ->
                          _ = put_callid(CallId),
                          erlang:apply(Module, Function, Arguments)
                  end).
+spawn(Fun, Arguments) ->
+    CallId = get_callid(),
+    erlang:spawn(fun() ->
+                         _ = put_callid(CallId),
+                         erlang:apply(Fun, Arguments)
+                 end).
```

Comment:
please create a ticket for this, "replace usage of wh_util spawn/3 with spawn/2 on all applications"